src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/ReportResource.kt

```diff
@@ -84,10 +84,29 @@ class ReportResource(
     ],
   )
   fun getBasicReports(
+    @Parameter(
+      description = "Filter by given locations, typically prison IDs",
+      example = "LEI,MDI",
+      array = ArraySchema(
+        schema = Schema(example = "MDI", minLength = 2, maxLength = 20),
```

Comment:
Not sure what the limit should be ! - 20 might be too small - Think we should allow about 150 (e.g. all prisons )

Reply:
this schema is not for the list itself